.evg.yml

```diff
@@ -3,100 +3,196 @@ functions:
+          chmod +x libmongo.so
+
+          echo "building transpiler"
+          cd src/github.com/10gen/stitch/etc/transpiler
+          yarn install && yarn run build -t ${transpiler_target}
+    - command: shell.exec
+      params:
+        shell: "bash"
+        background: true
+        script: |
+          set -e
+          export GOPATH=`pwd`
+          export PATH="`pwd`:$PATH"
+          export STITCH_PATH=$GOPATH/src/github.com/10gen/stitch
+          export PATH="$PATH:$STITCH_PATH/etc/transpiler/bin"
+          echo "running stitch"
+          # TODO: Probably better to get an API key added to mimic cloud
+          go run $STITCH_PATH/cmd/auth/user.go addUser -domainID 000000000000000000000000 -mongoURI mongodb://localhost:26000 -salt 'DQOWene1723baqD!_@#' -id "unique_user@domain.com" -password "password"
+          go run $STITCH_PATH/cmd/server/main.go --configFile $STITCH_PATH/etc/configs/test_config.json
+    - command: shell.exec
+      params:
+        shell: "bash"
+        script: |
+          counter=0
+          until $(curl --output /dev/null --silent --head --fail http://localhost:9090); do
+            echo "checking for API server to be up..."
+            sleep 1
+            let counter++
+            if [ $counter -gt 100 ]; then
+              exit 1
+            fi
+          done
+  "publish_codecov":
+    - command: shell.exec
+      params:
+        shell: "bash"
+        silent: true
+        script: |
+          set -e
+          echo export CODECOV_TOKEN=${codecov_token} >> creds
+    - command: shell.exec
+      params:
+        shell: "bash"
+        script: |
+          source ./creds
+          cd stitch-ios-sdk
+
+          if [ -z "${github_pr_number}" ]
+          then
+            if [ "${is_patch}" = "true" ]; then
+              exit 0
+            fi
+            bash <(curl -s https://codecov.io/bash) -C ${revision} -b ${build_id}
+          else
+            PR_INFO=`curl https://api.github.com/repos/${github_org}/${github_repo}/pulls/${github_pr_number}`
+            BRANCH=`echo $PR_INFO | python -c "import sys, json; print json.load(sys.stdin)['head']['ref']"`
+            bash <(curl -s https://codecov.io/bash) -B $BRANCH -C ${revision} -P ${github_pr_number} -b ${build_id}
+          fi
 tasks:
-  - name: github_pull_request_start
-    priority: 100
+  - name: lint
     commands:
+      - func: "fetch_source"
       - command: shell.exec
-        params:
+        params: 
+          shell: "bash"
           script: |
-            if ! [ "${is_patch}" = "true" ]; then
-              exit 0
-            fi
-            echo pending > commit_state
-      - func: "set_github_build_status"
-
-  - name: github_pull_request_end
-    depends_on:
-      - name: github_pull_request_start
-      - name: run_tests
-        status: "*"
-        variant: linux-64
+            set -e
+            cd stitch-ios-sdk/StitchCore
+            swiftlint
```

Comment:
[q] Can we run a script that lints everything, not just core? That's what java is doing